cibuildwheel/linux.py

```diff
@@ -217,6 +217,18 @@ def build(options: BuildOptions) -> None:
                         # clean up test environment
                         docker.call(['rm', '-rf', venv_dir])
 
+                    existing_output_files = docker.call(['ls'], cwd=container_output_dir).split('\n')
+                    for repaired_wheel in repaired_wheels:
+                        if repaired_wheel.name in existing_output_files:
+                            message = f'Created wheel ({repaired_wheel}) already exists in output directory.'
+                            if 'abi3' in repaired_wheel.name:
+                                message += ('\n'
+                                            "It looks like you are created wheels against Python's limited API/stable ABI;\n"
```

Comment:
"Created" or "are creating"

Reply:
Argh, right.

Reply:
> "Created" or "are creating"

Or "are building", just to be annoying. But it is `cibuildwheel`, after all :-)